src/lm_buddy/jobs/_entrypoints/lm_harness.py

```diff
@@ -4,6 +4,7 @@
 import torch
 from lm_eval.models.huggingface import HFLM
 from lm_eval.models.openai_completions import OpenaiCompletionsLM
+from lm_eval.logging_utils import WandbLogger
```

Comment:
Do we need a version bump on `lm-eval` package to include this? Should update the pyproject.toml file?